### ISSUE TITLE
Warning/error are not accurate in new react compiler plugin

### ISSUE THREAD

**Issue by LianaHus** [ethereum/remix-project#1452]
try to compile this code
```
// SPDX-License-Identifier: GPL-3.0
pragma solidity >=0.7.0 <0.9.0;

contract Storage {
    uint256 number, s;
}
```

error is shown right on vertical icon notification badge but not in a error/warning output list of plugin

or here

**Comment by ryestew** [comment]
Let's discuss this one when planing the 0.19.0 release:  @joeizang, @ioedeveloper, @LianaHus

**Comment by joeizang** [comment]
I just tried to replicate the error @LianaHus mentioned and I am not sure if its the same but I have experienced some buggy behaviour. I tried to compile Storage.sol and I removed a few semi colons and braces. The icon on the vertical Icon panel only showed the spinner like its compiling but never got to show an error. When I fixed the errors back (missing semi colons and braces), the spinner on the compile button was moving up and down and not spinning and the button was disabled. After a long while it stopped and then the button became clickable and then shows a false compile time error in the solidity file.
